cmd/client/lib/cli.go

```diff
@@ -29,7 +32,7 @@ var (
 	URLFlag = &cli.StringSliceFlag{
 		Name:    "url",
 		Usage:   "root URL(s) for fetching randomness",
-		Aliases: []string{"http-failover"}, // DEPRECATED
+		Aliases: []string{"http-connect", "http-failover"}, // DEPRECATED
```

Comment:
where were we using http-connect?

Reply:
In the gossip relay. Although it's currently not working (which is what this PR is trying to fix) so we can probably just remove the alias.